src/Views/Form/Templates/Sequoia/assets/css/form.scss

```diff
@@ -103,6 +103,13 @@ p {
 	}
 }
 
+.advance-btn:disabled,
+.give-submit:disabled,
+.download-btn:disabled {
+	filter: grayscale(0.6);
```

Comment:
@henryholtgeerts let's reduce this so it's not THAT gray.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/60882b2de3164296e0dd1f7f0734f89f6259f1ff